hugr-core/src/hugr/views/sibling_subgraph.rs

```diff
@@ -778,22 +776,36 @@ mod tests {
         }
     }
 
+    /// A Module with a single function from three qubits to three qubits.
+    /// The function applies a CX gate to the first two qubits and a Rz gate (with a constant angle) to the last qubit.
     fn build_hugr() -> Result<(Hugr, Node), BuildError> {
         let mut mod_builder = ModuleBuilder::new();
         let func = mod_builder.declare(
             "test",
             Signature::new_endo(type_row![QB_T, QB_T, QB_T])
-                .with_extension_delta(test_quantum_extension::EXTENSION_ID)
+                .with_extension_delta(ExtensionSet::union_over([
```

Comment:
I'm not sure what's missing, but the compiler doesn't like that (From != FromIter) and after formatting it ends up being longer.

I changed the `::union_over` for a `::from_iter` though, since the former creates a new ExtensionSet for each id before `extend`ing the final one's set.